sqlc.yaml

```diff
@@ -0,0 +1,10 @@
+version: "2"
+sql:
+  - engine: "sqlite"
```

Comment:
See the `tapd` config, we can get by with just the `postgres` engine. Then we also don't need to segment things for sqlite vs not, can just live under a `sqlc` folder.